Sources/KlaviyoCore/AppLifeCycleEvents.swift

```diff
@@ -9,18 +9,27 @@ import Combine
 import Foundation
 import UIKit
 
-enum LifeCycleErrors: Error {
+public enum LifeCycleErrors: Error {
     case invalidReachaibilityStatus
 }
 
-struct AppLifeCycleEvents {
-    var lifeCycleEvents: () -> AnyPublisher<KlaviyoAction, Never> = {
+public enum LifeCycleEvents {
+    case terminated
+    case forgrounded
```

Comment:
Don't be sorry. It's my bad that I can't spell to save my life! 

Fixed it and pushed. Didn't commit your changes since I made some locally and didn't want to deal with pulling and pushing. Thanks for the commit suggestions nonetheless.